inst/stan/wwinference.stan

```diff
@@ -190,15 +227,49 @@ transformed parameters {
   // Site level disease dynamic estimates!
   i0_site_over_n = inv_logit(logit(i0_over_n) + eta_i0 * sigma_i0);
   growth_site = initial_growth + eta_growth * sigma_growth; // site level growth rate
+
+  // Site level spatial Rt------------------------------------------------------
+  if (ind_corr_func == 1){
+    // If no dist matrix given, use n_sites + 1 = n_subpops were all ind.
+    non_norm_omega = independence_corr_func(n_subpops - 1);
+  }
+  else {
+    non_norm_omega = exponential_decay_corr_func(norm_dist_matrix, phi, l);
+  }
+  norm_omega = matrix_normalization(non_norm_omega);
```

Comment:
```suggestion
    non_norm_omega = independence_corr_func(n_subpops - 1);
    norm_omega = non_norm_omega;
  }
  else {
    non_norm_omega = exponential_decay_corr_func(norm_dist_matrix, phi, l);
    norm_omega = matrix_normalization(non_norm_omega);
  }
```

Reply:
Since the independence correlation function output is already determinant normalized, suggest skipping the normalization call when we use it.